packages/cluster-ui/src/util/appStats.ts

```diff
@@ -20,7 +20,7 @@ export function stdDev(stat: NumericStat, count: number) {
 }
 
 export function stdDevLong(stat: NumericStat, count: number | Long) {
-  return stdDev(stat, Number(FixLong(count)));
+  return !!stat && stdDev(stat, Number(FixLong(count)));
```

Comment:
@elkmaster, it looks like `null` validation has to be handled inside of `variance` function. It will ensure that following functions (`variance`, `stdDev`, and `stdDevLong`) won't fail with this error.

```javascript
export function variance(stat: NumericStat, count: number) {
  return (stat?.squared_diffs || 0) / (count - 1);
}
```